setuptools/msvc9_support.py

```diff
@@ -57,6 +58,10 @@ def patch_for_specialized_compiler():
     Microsoft Visual C++ 14.0:
         Microsoft Visual C++ Build Tools 2015 (x86, x64, arm)
     """
+    if platform.system() != 'Windows':
```

Comment:
I did this to help pass test on Linux.
If related files are missing on other platform, I am OK it is redundant. If files are present, patched them if they are not used is not useful.